modin/experimental/xgboost/test/test_dmatrix.py

```diff
@@ -74,67 +75,49 @@ def test_dmatrix_feature_names_and_feature_types(data, feature_names, feature_ty
     check_dmatrix(data, feature_names=feature_names, feature_types=feature_types)
 
 
-@pytest.mark.parametrize(
-    "feature_names",
-    [
-        ["Feature1", "Feature2", "Feature3", "Feature4", "Feature5"],
-        [u"??1", u"??2", u"??3", u"??4", u"??5"],
-    ],
-)
-def test_feature_names(feature_names):
-    data = np.random.randn(100, 5)
-    label = np.array([0, 1] * 50)
+def test_feature_names():
+    dataset = load_breast_cancer()
+    X = dataset.data
+    y = dataset.target
+    feature_names = [f"feat{i}" for i in range(X.shape[1])]
 
     check_dmatrix(
-        data,
-        label,
+        X,
+        y,
         feature_names=feature_names,
     )
 
-    dm = xgb.DMatrix(data, label=label, feature_names=feature_names)
-    md_dm = mxgb.DMatrix(
-        pd.DataFrame(data), label=pd.Series(label), feature_names=feature_names
+    dmatrix = xgb.DMatrix(X, label=y, feature_names=feature_names)
+    md_dmatrix = mxgb.DMatrix(
+        pd.DataFrame(X), label=pd.Series(y), feature_names=feature_names
     )
 
     params = {
-        "objective": "multi:softprob",
+        "objective": "binary:logistic",
         "eval_metric": "mlogloss",
-        "eta": 0.3,
-        "num_class": 3,
     }
 
-    bst = xgb.train(params, dm, num_boost_round=10)
-    md_bst = mxgb.train(params, md_dm, num_boost_round=10)
+    booster = xgb.train(params, dmatrix, num_boost_round=10)
+    md_booster = mxgb.train(params, md_dmatrix, num_boost_round=10)
 
-    predictions = bst.predict(dm)
-    modin_predictions = md_bst.predict(md_dm)
+    predictions = booster.predict(dmatrix)
+    modin_predictions = md_booster.predict(md_dmatrix)
 
-    preds = np.asarray([np.argmax(line) for line in predictions])
-    md_preds = np.asarray([np.argmax(line) for line in modin_predictions.to_numpy()])
+    preds = pd.DataFrame(predictions).apply(lambda x: round(x))
```

Comment:
```suggestion
    preds = pandas.DataFrame(predictions).apply(lambda x: round(x))
```
I would prefer to be explicit so let's specify `axis` parameter for `apply`. Why do we need `round`?

Reply:
also why don't you directly `.apply(round)` or `.apply(np.round)`?

Reply:
@YarShev, `axis` providing is added (I don't know if this makes sense, because we make `apply` for 1 column dataframe using non-reduce callable). `round` is need to determine class label (0 or 1) for each prediction value from column.
@vnlitvinov, I agree, `np.round` is used instead of lambda.

Reply:
I would also use pandas DataFrame for non Modin data here as I suggested earlier.

Reply:
No see the need in this, because in this test we don't compare pandas/modin dataframes functionality, we just use dataframe to apply `round`. But I don't mind, applied

Reply:
If we had a bug in `apply`, we might have tests passed and wouldn't see an actual problem (even though we test `apply` separately).